src/core/ext/filters/channel_idle/channel_idle_filter.cc

```diff
@@ -294,6 +294,7 @@ const grpc_channel_filter MaxAgeFilter::kFilter =
     MakePromiseBasedFilter<MaxAgeFilter, FilterEndpoint::kServer>("max_age");
 
 void RegisterChannelIdleFilters(CoreConfiguration::Builder* builder) {
+  GPR_ASSERT(MaxAgeFilter::kFilter.init_call != nullptr);
```

Comment:
Do we want to keep this line? Or this is just for debugging?

Reply:
Let's keep it for now, there's little runtime cost and it's a good sanity check